x-pack/plugins/encrypted_saved_objects/server/saved_objects/index.test.ts

```diff
@@ -47,14 +47,14 @@ describe('#setupSavedObjects', () => {
     setupContract = setupSavedObjects({
       service: mockEncryptedSavedObjectsService,
       savedObjects: coreSetupMock.savedObjects,
-      security: securityMock.createSetup(),
       getStartServices: coreSetupMock.getStartServices,
     });
   });
 
   describe('#setupContract', () => {
     it('includes hiddenTypes when specified', async () => {
-      await setupContract({ includedHiddenTypes: ['hiddenType'] });
+      setupContract({ includedHiddenTypes: ['hiddenType'] });
+      await nextTick();
```

Comment:
Minor cleanup here: `setupContract`: it's confusing to `await setupContract` as that function doesn't return a promise. But a promise does get initialized in that function, and the test depends on that promise being resolved.